e2e/test/scenarios/onboarding/setup/setup.cy.spec.ts

```diff
@@ -12,6 +12,7 @@ import {
   resetSnowplow,
```

Comment:
Stres test of the e2e: https://github.com/metabase/metabase/actions/runs/10422796666/job/28868095934